parcels/field.py

```diff
@@ -21,7 +21,7 @@
     assert_valid_gridindexingtype,
     assert_valid_interp_method,
 )
-from parcels.tools._helpers import deprecated_made_private
+from parcels.tools._helpers import deprecated_made_private, pretty_field
```

Comment:
Not sure these names are as meaningful as they could be. Why not `prettyprint_<class>` (or `prettyrepr_<class`), to make it more explicit it's about printing?